### PR TITLE
Update args dependency

### PR DESCRIPTION
**Pull request by kaendfinger**
cc @devoncarew 

After this, a release should probably be published, because of latest release of Dart.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ environment:
   sdk: '>=1.0.0 <2.0.0'
 
 dependencies:
-  args: '>=0.10.0 <0.13.0'
+  args: '>=0.12.0 <0.14.0'
```
Comment (devoncarew) [inline pubspec.yaml]: I'm assuming we do require at least `0.12.0`? I'd like to keep the constraints as broad as possible.

Comment (kaendfinger) [inline pubspec.yaml]: Ok, I'll expand it, because we don't require 0.12.

**Comment by kaendfinger** [comment]
@devoncarew Fixed

**Comment by devoncarew** [comment]
Thanks!

**Pull request merged**